config/routes.rb

```diff
@@ -1,2 +1,3 @@
 Rails.application.routes.draw do
+  root 'reviews#index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.